build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e
+
+${CARMA_OPT_DIR}/scripts/build_script.sh -p $@ -DBUILD_TESTS=ON -DCREATE_DEB_PACKAGE=ON
```

Comment:
Assume this build_script.sh is from carma-build repo, where in the process we download the carma-build and install it in the ${CARMA_OPT_DIR} directory?